tests/test_items.py

```diff
@@ -97,9 +97,9 @@ def test_get_item__failed(self):
         resp = self.app.get('/items/{item_id}'.format(item_id=WRONG_UUID))
         assert resp.status_code == client.NOT_FOUND
 
-    def test_put_item__success(self):
+    def test_patch_item__success(self):
         item = Item.create(**TEST_ITEM)
-        resp = self.app.put('/items/{item_id}'.format(item_id=item.item_id),
+        resp = self.app.patch('/items/{item_id}'.format(item_id=item.item_id),
                             data=json.dumps(TEST_ITEM2),
```

Comment:
PEP8 in queste righe. Se vai a capo in questo modo dopo la virgola della riga 102 la prima lettera deve essere in linea con l'apertura della parentesi la riga sopra.

https://www.python.org/dev/peps/pep-0008/#indentation